Cargo.toml

```diff
@@ -15,11 +15,11 @@ objc = "0.2"
 objc_id = "0.1"
 objc-foundation = "0.1"
 
-[target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="emscripten"))))'.dependencies]
+[target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="ios", target_os="emscripten"))))'.dependencies]
```

Comment:
All these annotations seem absolutely massive.

I feel like we could simplify them a lot, by just making it either linux or bsd.

Reply:
There're a lot of bsd and some solaris, etc. I don't think that we can do anything about it.

Reply:
For more information look there https://github.com/rust-windowing/winit/blob/master/Cargo.toml

Reply:
Hm, I guess that's a fair point. Well it's not like that's going to break anything, so it should be fine. Let's just keep this change simple.